templates/zerver/help/join-a-zulip-organization.md

```diff
@@ -0,0 +1,40 @@
+# Join a Zulip Organization
+
+An **organization**  is a private chamber hosted on Zulip with its own users,
+ streams, customizations, etc. Administrators manage the organization,
+ controlling the specific settings of the organizations
+ that they own.
+
+1. To join a Zulip organization, first navigate to the URL of the Zulip server.
+
+2. Click the register button in the top right hand corner of the screen.
+
+    ![Register button](/static/images/help/register-button.png)
+
+3. You will be taken to the register page and prompted to **Enter your
+ work email address**. Enter your work email address and click **Sign Up**.
+
+
+4. You will then be taken to the page shown below, which asks you to confirm your
+ email address. Wait for the email to arrive in your inbox,
+ and then click on the confirmation link in the email to confirm your email
+ address.
+
+    ![Email Confirmation](/static/images/help/confirm-email.png)
+
+    !!! tip ""
+        If the email doesn't arrive or you misplace the email, you can request
+         another confirmation link by clicking on the **Resend** link.
+
+5. Finally, when you click the confirmation link, you will be taken
```

Comment:
Change this to `Finally, when you click the confirmation link, you will be prompted to..."